specification/automation/resource-manager/Microsoft.Automation/preview/2017-05-15-preview/softwareUpdateConfiguration.json

```diff
@@ -126,33 +126,33 @@
         "operationId": "SoftwareUpdateConfigurations_GetByName",
         "parameters": [
           {
-            "$ref": "./definitions.json#/parameters/SubscriptionIdParameter"
+            "$ref": "../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
           },
           {
-            "$ref": "./definitions.json#/parameters/ResourceGroupNameParameter"
+            "$ref": "../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
           },
           {
-            "$ref": "./definitions.json#/parameters/AutomationAccountNameParameter"
+            "$ref": "../../common/v1/definitions.json#/parameters/AutomationAccountNameParameter"
           },
           {
-           "name": "softwareUpdateConfigurationName",
+            "name": "softwareUpdateConfigurationName",
             "description": "The name of the software update configuration to be created.",
             "type": "string",
             "required": true,
             "in": "path"
           },
           {
-            "$ref": "./definitions.json#/parameters/ApiVersionParameter"
+            "$ref": "../../common/v1/definitions.json#/parameters/ApiVersionParameter"
           },
           {
-            "$ref": "./definitions.json#/parameters/clientRequestId"
+            "$ref": "../../common/v1/definitions.json#/parameters/clientRequestId"
           }
         ],
         "responses": {
           "200": {
             "description": "A single software update configuration.",
             "schema": {
-              "$ref": "./definitions.json#/definitions/softwareUpdateConfiguration"
+              "$ref": "#/definitions/softwareUpdateConfiguration"
             }
           },
           "404": {
```

Comment:
status codes "404" for operation "SoftwareUpdateConfigurations_GetByName" were present in the swagger spec, however they were not present in x-ms-examples.

Reply:
Fixed.